docs/dapps/smart-contracts/using-foundry-with-the-avalanche-c-chain.md

```diff
@@ -0,0 +1,158 @@
+# Using Foundry with the Avalanche C Chain
+
+## Introduction
+
+This article shows how to deploy and interact with smart contracts using foundry on the [Fuji C-Chain](../../quickstart/fuji-workflow.md).
+
+[Foundry](https://book.getfoundry.sh/) is a smart contract development toolchain written in Rust.
+
+Foundry manages your dependencies, compiles your project, runs tests, deploys, and lets you interact with the chain from the command-line.
+
+## Prerequisites
+
+- You have installed [foundry](https://book.getfoundry.sh/getting-started/installation.html)
+- You are familiar with [Avalanche Smart Contract Quickstart](https://github.com/ava-labs/avalanche-smart-contract-quickstart)
+
+## Getting Started
+
+This section will walk you through creating an ERC721 with Foundry and Avalanche Smart Contract Quickstart.
+
+Clone the Avalanche Smart Contract Quickstart repo and install its dependencies by running:
+
+```zsh
+git clone https://github.com/ava-labs/avalanche-smart-contract-quickstart.git
+cd avalanche-smart-contract-quickstart
+yarn
+```
+
+In order to deploy contracts, you need to have some AVAX. You can use a pre-funded account on a local network or get testnet AVAX from the [Avalanche Faucet](https://faucet.avax.network), which is an easy way to get to play around with Avalanche. After getting comfortable with your code, you can run it on Mainnet after making the necessary changes to your workflow.
```

Comment:
This article is for Fuji. There is no prefunded account on Fuji.

Reply:
roger that ty